v0/destinations/active_campaign/transform.js

```diff
@@ -219,29 +210,25 @@ const customTagProcessor = async (message, category, destination) => {
   // Ref - https://developers.activecampaign.com/reference#create-contact-tag
   await Promise.all(
     tagIds.map(async tagId => {
-      try {
-        res = await axios.post(
-          `${destination.Config.apiUrl}${
-            category.mergeTagWithContactUrl
-              ? category.mergeTagWithContactUrl
-              : ""
-          }`,
-          {
-            contactTag: {
-              contact: createdContact.id,
-              tag: tagId
-            }
-          },
-          {
-            headers: {
-              "Content-Type": "application/json",
-              "Api-Token": destination.Config.apiKey
-            }
-          }
-        );
-      } catch (err) {
-        errorHandler(err, "Failed to merge created contact with created tags");
-      }
+      endpoint = `${destination.Config.apiUrl}${category.mergeTagWithContactUrl}`;
+      requestData = {
+        contactTag: {
+          contact: createdContact.id,
+          tag: tagId
+        }
+      };
+      requestOptions = {
+        headers: {
+          "Content-Type": "application/json",
+          "Api-Token": destination.Config.apiKey
+        }
+      };
+      res = httpPOST(endpoint, requestData, requestOptions);
```

Comment:
```suggestion
      res = await httpPOST(endpoint, requestData, requestOptions);
```

Reply:
Updated.

Reply:
Not required as we are using Promise.all